docs/release-notes/latest-changes.md

```diff
@@ -16,6 +16,7 @@ Portions created by Andreas Wacknitz are Copyright (C) 2018.
 Portions created by Aurelien Larcher are Copyright (C) 2018.
 Portions created by Ken Mays are Copyright (C) 2014.
 Portions created by Michal Nowak are Copyright (C) 2020.
```

Comment:
If these contributions are to be removed, then the authors should probably also be removed.

Reply:
> I'm not sure about removing the old changes.
Hopefully they are not removed, I'll check, ...hmm, oops!
We have a dilemma: 'latest-changes' should be latest changes, and not going back 10 years or so---maybe going back as far as the previous ISO release (or whatever!). But we shouldn't throw away the old stuff either; maybe
move the older stuff to archive-releases? I could add this to the PR.
> The new website looks to have the information already...
Where? I cannot find detailed release notes there, except for general stuff. (Can you correct me if I' wrong.)
Generally, release notes are based on PRs, which are taken from the code, which is here; so the new Website would only provide a link to here.
Anyway, whether here, there or somewhere else; someone is going to have to generate and document the changes, which will have to be done by someone and I doubt it whether there are too many show of hands.

Release notes are important to any project, and we don't have any and I think we should do something about it, which is why I made this PR.
I've also commented in the ticket that the best way to automate this is for each PR to include a keyword, i.e., REL somewhere in the title of the PR, so that PR can be included in release notes (a lot of PRs are just forced re-builds or whatnot and should not 'clutter' the release notes. It would then be easy to parse only those PRs with the keyword to build the release notes.

So how to proceed?

Reply:
The detailed log section of the [Hipster 2023.04 Announcement](https://www.openindiana.org/announcements/openindiana-hipster-2023-04-announcement/) contains a lot of information on version upgrades. Much more than is in this PR. This seems odd as I would expect this to be the same. There is a need for a release notes section, but I'm not sure either is adequate currently.

So far as latest changes vs old changes, there could be a separate page with all of the old changes. With this page containing only the latest changes?